Source/WebCore/page/PartitionedSecurityOrigin.h

```diff
@@ -51,6 +51,8 @@ struct PartitionedSecurityOrigin {
     bool isHashTableDeletedValue() const { return topOrigin.isHashTableDeletedValue(); }
     bool isHashTableEmptyValue() const { return topOrigin.isHashTableEmptyValue(); }
 
+    PartitionedSecurityOrigin isolatedCopy() const & { return { topOrigin->isolatedCopy(), clientOrigin->isolatedCopy() }; }
```

Comment:
Why the ` & ` since there is no ` && ` version?

Reply:
Fixed